### PR TITLE
Compose

### PR DESCRIPTION
**Pull request by jeff-hernandez**
This PR updates the tutorial by using Compose to create labels. Tutorial is located [here](https://github.com/Featuretools/predict-next-purchase/blob/compose/Tutorial.ipynb).

### CODE REVIEW

File: utils.py [hunk 1]
```diff
@@ -3,11 +3,20 @@
 import numpy as np
 import os
 
-def load_entityset(data_dir):
-    order_products = pd.read_csv(os.path.join(data_dir, "order_products__prior.csv"))
-    orders = pd.read_csv(os.path.join(data_dir, "orders.csv"))
-    departments = pd.read_csv(os.path.join(data_dir, "departments.csv"))
-    products = pd.read_csv(os.path.join(data_dir, "products.csv"))
+def load_order_products(data_dir, nrows=None):
```
Comment (kmax12) [inline utils.py]: does this method get used?

File: Tutorial.ipynb [hunk 2]
```diff
@@ -1172,42 +1272,46 @@
...
+   "execution_count": 20,
    "metadata": {},
    "outputs": [
     {
      "name": "stderr",
      "output_type": "stream",
      "text": [
       "[Parallel(n_jobs=1)]: Using backend SequentialBackend with 1 concurrent workers.\n",
-      "[Parallel(n_jobs=1)]: Done   3 out of   3 | elapsed:  4.5min finished\n"
+      "/Users/jeff.hernandez/miniconda/envs/notebook/lib/python3.7/site-packages/joblib/externals/loky/process_executor.py:706: UserWarning: A worker stopped while some jobs were given to the executor. This can be caused by a too short worker timeout or by a memory leak.\n",
```
Comment (kmax12) [inline Tutorial.ipynb]: can we get rid of this error in output>

**Comment by kmax12** [comment]
noticed one other thing. there is a line in the notebook

```
training_window=ft.Timedelta("60 days"), # same as above
```

I don't think the reference to "above" is valid anymore.

**Comment by kmax12** [comment]
also, looks like the number of label times went way up, increasing the overall runtime. can we look into keep the runtime about the same as it was before?

File: utils.py (hunk 1 above)
Comment (jeff-hernandez) [reply utils.py]: No, this method doesn't get used. The `make_labels` and `dask_make_labels` also don't get used. I can remove these methods.

File: Tutorial.ipynb (hunk 2 above)
Comment (jeff-hernandez) [reply Tutorial.ipynb]: Removed the error.

**Comment by jeff-hernandez** [comment]
Brought the runtime lower and closer to the previous one.

File: README.md [hunk 3]
```diff
@@ -13,7 +13,7 @@ In this tutorial, we show how [Featuretools](https://www.featuretools.com) can b
 ## Highlights
 
 * We automatically generate 150+ features using Deep Feature Synthesis and select the 20 most important features for predictive modeling
-* We build a pipeline that it can be reused for numerous prediction problems (you can try this yourself!)
+* We automatically generate label times using [Compose](https://compose.featurelabs.com) which can be reused for numerous prediction problems (you can try this yourself!)
```
Comment (kmax12) [inline README.md]: let's link to the github instead. here and in the notebook

**Pull request merged**